connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -314,6 +356,23 @@ private <T extends SourceConnector> void awaitConnectorTasksStart(final MirrorMa
         }, MM_START_UP_TIMEOUT_MS, "Tasks for connector " + clazz.getSimpleName() + " for MirrorMaker instances did not transition to running in time");
     }
 
+    private <T extends SourceConnector> void awaitConnectorConfiguration(MirrorMaker mm, Class<T> clazz, SourceAndTarget sourceAndTarget, Predicate<Map<String, String>> predicate) throws InterruptedException {
+        String connName = clazz.getSimpleName();
+        waitForCondition(() -> {
+            try {
+                FutureCallback<Map<ConnectorTaskId, Map<String, String>>> cb = new FutureCallback<>();
+                herder(mm, sourceAndTarget).tasksConfig(connName, cb);
```

Comment:
Interesting call checking on task configs instead of the connector config. Just out of curiosity, why one instead of the other?

(Not a blocker)

Reply:
I think I was doing a sort of "end-to-end-test" that verified that both the new configuration got applied to the connector, and that internal forwarding took place for the task configurations. I realize now that the method name doesn't match this implementation, so i renamed it to match.

Reply:
Ah, very nice. Good call on end-to-end testing 👍